utilities/transactions/transaction_lock_mgr.cc

```diff
@@ -643,26 +643,27 @@ void TransactionLockMgr::UnLock(PessimisticTransaction* txn,
 }
 
 void TransactionLockMgr::UnLock(const PessimisticTransaction* txn,
-                                const TransactionKeyMap* key_map, Env* env) {
-  for (auto& key_map_iter : *key_map) {
-    uint32_t column_family_id = key_map_iter.first;
-    auto& keys = key_map_iter.second;
-
-    std::shared_ptr<LockMap> lock_map_ptr = GetLockMap(column_family_id);
+                                const LockTracker& tracker, Env* env) {
+  std::unique_ptr<LockTracker::ColumnFamilyIterator> cf_it(
+      tracker.GetColumnFamilyIterator());
+  if (!cf_it) return;
```

Comment:
Maybe there should be an assertion.

Reply:
The design was that if point lock is not used, then `GetColumnFamilyIterator()` might return `nullptr`.
But after second thought, I forced the API to never return `nullptr`, if there is no column family to iterate, just return an empty iterator.

Reply:
I see, I still feel like we should add a debug assertion somewhere, since this is something that's not supposed to happen.

Reply:
already added `assert(cf_it != nullptr)`